README.md

```diff
@@ -596,14 +595,14 @@ New features and bug fixes are released on the latest major version of the `stri
 
 ## Development
 
-Run all tests:
+Run all tests (installing the dependencies first, if needed)
 
 ```bash
-$ yarn install
-$ yarn test
+$ just install
+$ just test
 ```
 
-If you do not have `yarn` installed, you can get it with `npm install --global yarn`.
+If you do not have `yarn` installed, consult its [installation instructions](https://classic.yarnpkg.com/lang/en/docs/install/).
```

Comment:
this ensures people can follow whatever the latest guidance is (like installing from a non-js package manager)